pkg/api/mlflow/config/auth/config_loader.go

```diff
@@ -0,0 +1,105 @@
+package auth
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/rotisserie/eris"
+	"gopkg.in/yaml.v3"
+)
+
+// UserPermissions represents permission object into which the user configuration is parsed.
+type UserPermissions struct {
+	data map[string]map[string]struct{}
+}
+
+// HasAccess makes check that user has permission to access to the requested namespace.
+func (p UserPermissions) HasAccess(namespace string, authToken string) bool {
```

Comment:
should this be in file "config_loader"? seems more related to later phase (authorizing requests)

Reply:
i wouldn't expect `HasAccess` in file `config_loader.go` -- seems like it belongs in file `auth.go`